src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPageFairRecoveryOasis.java

```diff
@@ -35,12 +34,10 @@ public void adsRecoveryPageFairOasis() {
     // when PF recovered ad is on page, inserts span elements as a direct children of body
     adsRecoveryObject.wait.forElementPresent(AdsRecoveryObject.PF_RECOVERED_ADS_SELECTOR);
 
-    // verify that adblock is turned on on that page
-    adsRecoveryObject.verifyNoAdsOnPage();
-
+    // BLB is always present but hidden by PageFair after triggering it we just have to wait a while (7s)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)